gsplat/cuda_legacy/_wrapper.py

```diff
@@ -300,6 +293,10 @@ def rasterize_gaussians(
     block_width: int,
     background: Optional[Float[Tensor, "channels"]] = None,
     return_alpha: Optional[bool] = False,
+    model_type: Literal["3dgs", "2dgs"] = "3dgs",
```

Comment:
I think it's better to just make a separate function for 2dgs, but @liruilong940607 what do you think?